### DISCUSSION TITLE
Upgrade from 1.0 to 3.0  - how does it work?

### DISCUSSION THREAD

**Discussion by schebitz** [payloadcms/payload#10007] [Q&A] [answered]
I am using payload 1.0 for a website. Now I think about upgrading directly to the latest version. How will that work?

**Comment by tyteen4a03** [comment, marked as answer]
You might be better off recreating your config in 3.0, then work on a migration script to port the data over. (I also recommend the same for any non-trivial 2.x apps)

**Reply by schebitz** [reply]
Thx a lot for that tipp which I will discuss with our developers. Did you do that already?

**Reply by tyteen4a03** [reply]
> Thx a lot for that tipp which I will discuss with our developers. Did you do that already?

No, but I attempted an upgrade from V2 to V3 beta (in the later stages) and didn't enjoy it. I think it's easier to start over.

**Comment by schebitz** [comment]
Actually, that’s exactly what I feared.

Timothy Choi ***@***.***> schrieb am Di. 17. Dez. 2024 um
00:02:
